test/cmdlineTests/external_function_pointers_taking_calldata_args/input.json

```diff
@@ -0,0 +1,13 @@
+{
+    "language": "Solidity",
+    "sources": {
+        "url_not_found.sol": {
+            "urls": [
+                "/projects/contracts.sol",
+                "contract.sol",
+                "/tests/test.sol"
```

Comment:
Please also add another test that has 3 files that do not exist. Looks like in that case we do emit 3 errors and we should have it covered too.

Reply:
Adding test case in the latest change.